metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name":    "simp-cron",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
In this case, I don't think so. It's an additional parameter that has it's value sourced from `simplib::lookup`, and it doesn't need to be used by a user.